src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/DefaultJsonTypeInfoResolver.Converters.cs

```diff
@@ -39,7 +39,7 @@ private static JsonConverterFactory[] GetDefaultFactoryConverters()
 
         private static Dictionary<Type, JsonConverter> GetDefaultSimpleConverters()
         {
-            const int NumberOfSimpleConverters = 26;
+            const int NumberOfSimpleConverters = 28;
```

Comment:
Should this one be ifdef'ed as well?

Reply:
It could be but it will be unnecessarily over-complicated. I generally agree with @eiriktsarpalis's response here: https://github.com/dotnet/runtime/pull/69160/files#r871454257